test/scenarios-periodics/el92-src@rpm-standard1.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+scenario_create_vms() {
+    prepare_kickstart host1 kickstart-liveimg.ks.template ""
+    launch_vm host1 "rhel-9.2-microshift-source"
+    configure_vm_firewall host1
+    run_command_on_vm host1 "sudo systemctl enable --now microshift"
```

Comment:
I thought we were enabling microshift service in the source base template. Are we losing this setting somehow?
If so, should we also enable the `microshift-test-agent` service here?

Reply:
The blueprint item to enable the service does not apply to the RPM-based system.

The tests do not use the agent service, so it's not being enabled.